docs/index.rst

```diff
@@ -2,42 +2,64 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
+``sunpy`` is a community-developed, free and open-source solar
 data analysis environment for Python.
 
+New Users
```

Comment:
I think this would be better at one title level lower, to make sure "sunpy core Documentation" stays the biggest at the top.